python/pyspark/sql/tests.py

```diff
@@ -5156,6 +5156,15 @@ def test_retain_group_columns(self):
             expected1 = df.groupby(df.id).agg(sum(df.v))
             self.assertPandasEqual(expected1.toPandas(), result1.toPandas())
 
+    def test_array_type(self):
```

Comment:
This is related, but I figured its shouldn't hurt to add an array test in GroupedAggPandasUDFTests..